{
  FalseIfNotAvailable function=new FalseIfNotAvailable();
  RuntimeException exception=new RuntimeException();
  try {
    function.create(exception);
  }
 catch (  Exception ex) {
    assertEquals(ex,exception);
  }
}
